backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/SchedulingConfig.java

```diff
@@ -0,0 +1,29 @@
+package de.bund.digitalservice.ris.caselaw.adapter;
+
+import javax.sql.DataSource;
+import net.javacrumbs.shedlock.core.LockProvider;
+import net.javacrumbs.shedlock.provider.jdbctemplate.JdbcTemplateLockProvider;
+import net.javacrumbs.shedlock.spring.annotation.EnableSchedulerLock;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.scheduling.annotation.EnableScheduling;
+
+/**
+ * Scheduler config with shedlock for distributed scheduling. When a cronjob is set up, it is
+ * ensured that multiple pods do not execute the same job in parallel.
+ */
+@Configuration
+@EnableScheduling
+@EnableSchedulerLock(defaultLockAtMostFor = "10m")
```

Comment:
Why ten minutes?
Is this really the right format?
```suggestion
@EnableSchedulerLock(defaultLockAtMostFor = "PT10M")
```

Reply:
I copied it from the setup page: https://github.com/lukas-krecan/ShedLock?tab=readme-ov-file#enable-and-configure-scheduled-locking-spring

I think the shedlock lib is more flexible with formats (and I find this more readable than the ISO format). The default scheduling of Spring does not seem to be so flexible with formats, but I can check again.

Reply:
The value set here is the default for all the locks. It must not be shorter than any job that is scheduled. I have set a shorter duration for our very quick job, but I think a shorter interval might lead to bugs later, if someone introduces a longer running job and forgets about changing this default here. Without a default, it would lock forever if a pod holding the lock crashes.

Reply:
Alright, thanks for your explanation. We can also discuss my questions in a call. Might be easier 😃